applications/ConvectionDiffusionApplication/tests/test_convection_diffusion_embedded_solver/ProjectParameters.json

```diff
@@ -38,9 +38,12 @@
             "element_name" : "EmbeddedLaplacianElement",
             "condition_name" : ""
         },
-	"linear_solver_settings" : {
-	    "solver_type" : "LinearSolversApplication.sparse_lu"
-	}
+        "linear_solver_settings" : {
+            "solver_type" : "LinearSolversApplication.sparse_lu"
```

Comment:
Doesn't it work with the default linear (iterative) solver?

Reply:
Yes, it does. I removed the linear_solver_settings.